js/header.js

```diff
@@ -1,9 +1,37 @@
-document.onscroll = function() {shrinkHeader()};
 header = document.getElementsByTagName('header')[0];
+
 function shrinkHeader() {
-    if (document.body.scrollTop > 50 && document.body.clientWidth >= 1200 || document.documentElement.scrollTop > 50 && document.body.clientWidth >= 1200) {
+    if ((document.documentElement.scrollTop || document.body.scrollTop) > 50 || document.body.clientWidth <= 1275) {
         header.className = 'shrink';
-    } else if (document.body.clientWidth >= 1200){ 
+    } else {
         header.className = '';
     }
 }
+
+//Shrink header on mobile and resized window
+body.onload = function () {
+    if (body.clientWidth <= 1275) {
+        header.className = 'shrink';
+    } else {
+        header.className = '';
+    }
+};
+body.onresize = function () {
+    if (body.clientWidth <= 1275) {
```

Comment:
Not necessarily a problem for right now, but might be good to declare some constants instead of all these magic numbers